sdk/runanywhere-kotlin/src/jvmAndroidMain/kotlin/com/runanywhere/sdk/foundation/bridge/extensions/CppBridgeLoraRegistry.kt

```
@@ -0,0 +1,130 @@
+            log(LogLevel.ERROR, "Failed to parse LoRA entry JSON: ${e.message}")
+            null
+        }
+    }
+
+    private fun parseLoraEntryArrayJson(json: String): List<LoraEntry> {
+        if (json == "[]" || json.isBlank()) return emptyList()
+        val entries = mutableListOf<LoraEntry>()
+        var depth = 0; var objectStart = -1
+        for (i in json.indices) {
+            when (json[i]) {
+                '{' -> { if (depth == 0) objectStart = i; depth++ }
```

detekt (1.23.8)

[warning] This else block is empty and can be removed.

(detekt.empty-blocks.EmptyElseBlock)